### ISSUE TITLE
Why do we hide the share button and grey it out? It makes me not want to click it cause it won't work

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapps-v2#2892]
**What problem is this design solving? Why is it necessary?**
*A clear and concise description of what the problem is. Link to any relevant issues, PRs, and other resources.*

We want people to share the project. Why is the share button grey'd out, I think it can be normal purple color or basically anything but grey

**Which metrics will define this feature's success?**
*Provide current and/or expected metrics.*

How many more sharing links we get posted on twitter (everyone is just posting the normal links still...)

**Additional context**
*Add any other context or screenshots related to your design here.*

![Screenshot 2023-07-11 at 8 10 37 PM](https://github.com/Giveth/giveth-dapps-v2/assets/14945613/99d79f98-e47c-4d57-9a76-c7807cefc30b)
![Screenshot 2023-07-11 at 8 10 46 PM](https://github.com/Giveth/giveth-dapps-v2/assets/14945613/124d8910-5d56-4d34-a350-51d572fe0e50)

**Comment by MoeNick** [comment]
@mosaeedi pls take a look

**Comment by mosaeedi** [comment]
Agree, it should be more prominent. I will use a darker shade of Gray.
I didn't use pink or purple here to avoid adding too many colors near the Donate button and to make sure the Donate button is the first thing users will notice.


<img width="414" alt="Screenshot 2023-07-13 at 1 19 48 PM" src="https://github.com/Giveth/giveth-dapps-v2/assets/13708322/10297ed6-6d4d-4919-b935-654e6706d1db">

@MohammadPCh 
I know the button component used the caps font style, but especially for this button, can we avoid using all caps font? 
The color for both the **like** and **share** buttons now is Gray/800.

**Comment by MohammadPCh** [comment]
It's fixed and now we have it on production.

**Comment by divine-comedian** [comment]
Great! Let's close this issue.
